### PR TITLE
Export interface and helper functions. (PT-186400442)

### PR DESCRIPTION
**Pull request by lublagg**

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -29,57 +29,4 @@ jobs:
...
-        with:
-          bucket: models-resources
-          prefix: starter-projects
-          awsAccessKeyId: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          awsSecretAccessKey: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          # Parameters to GHActions have to be strings, so a regular yaml array cannot
-          # be used. Instead the `|` turns the following lines into a string
-          topBranches: |
-            ["master"]
+        containers: [1, 2, 3]
```
Comment (dougmartin) [inline .github/workflows/ci.yml:32]: I don't think you need this file - there is no release process for this on s3.

File: webpack.config.js [hunk 2]
```diff
@@ -128,26 +27,10 @@ module.exports = (env, argv) => {
...
-      ...(DEPLOY_PATH ? [new HtmlWebpackPlugin({
-        filename: 'index-top.html',
-        template: 'src/index.html',
-        favicon: 'src/public/favicon.ico',
-        publicPath: DEPLOY_PATH
-      })] : []),
-      new CleanWebpackPlugin(),
-    ]
+    plugins: [],
+    externals: {
```
Comment (dougmartin) [inline webpack.config.js:31]: I don't think you need this externals - there is no usage of React in this package right?

**Review by dougmartin — COMMENTED** [review]
Looks like a good start.  I hope we can update Jonathan's code to remove as many of the `any`s as we can for type safety.

File: webpack.config.js (hunk 2 above)
Comment (lublagg) [reply webpack.config.js:31]: That's right, I'll remove them

**Pull request merged**